lib/client/js/state/Store.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * @class Store
+ */
+var Store = (function() {
+    /*
+     * @alias Store
+     * @constructor
+     * @param {Object} initialProps - A custom props object
+     * @returns {Store}
```

Comment:
I usually don't tag the return type, but a function itself returns the `this` implicitly when invoked with the `new` operator so it's valid JSDoc.

I'll remove it.